components/infobox/wikis/valorant/infobox_character_custom.lua

```diff
@@ -59,4 +59,13 @@ function CustomInjector:parse(id, widgets)
 	return widgets
 end
 
+---@param lpdbData table
+---@param args table
+---@return table
+function CustomCharacter:addToLpdb(lpdbData, args)
+	lpdbData.date = args.releasedate
+	lpdbData.extradata.releasedate = args.releasedate
```

Comment:
why the double storage?

Reply:
is there a consumer?
only thing i could find in template/module namespace that uses it is https://liquipedia.net/commons/Module:ChampionStats
and that seems to have no usage on val (unless used in some form) and could easily be fixed to support both ... `.date` and `.extradata.releasedate` (so we can easily progress infoboxes where necessary)

Reply:
I don't think we have any external consumers of datapoints to speak of, our data is just not good enough (yet) to warrant that :D